Code/application/Gui/Image/ImageFilterManager.cpp

```diff
@@ -320,11 +312,8 @@ vector<ImageFilterDescriptorImp*> ImageFilterManager::deserialize(const string&
                                  }
                               }
                            }
+                           // TODO JO test if shader program is valid
```

Comment:
If this TODO is still valid, we should add an issue to address this in the future. Otherwise remove the TODO